docs/user-guide/cli-ftpplugin.md

```diff
@@ -39,45 +39,26 @@ Use one of the following methods to install or update the plug-in:
 
 ## Creating a user profile
 
-You create a zftp profile to avoid entering your connection details each time that you issue a command. You can create multiple profiles and switch between them as needed. Use one of the following methods to create a profile:
+After you install the plug-in, create an FTP profile. An FTP profile is recommended to issue commands to via FTP. FTP profiles contain your host, port, user name, and password for connect to z/OS using FTP. You can create multiple profiles and switch between them as needed.
```

Comment:
There're some extra words/odd tenses here: "An FTP profile is recommended to issue commands **to via** FTP. FTP profiles contain your host, port, user name, and password **for connect** to z/OS using FTP."